tests/TestCase.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
I think you no need this file. Please check the LoginForm directly in test class

Reply:
I will do it at the end if no more classes are added.